web/src/components/admin/settings/queue/AdminQueueStats.vue

```diff
@@ -21,8 +21,7 @@
                 v-for="item in data"
                 :key="item.key"
                 class="h-full"
-                :class="`${item.color}`"
-                :style="{ width: `${item.percentage}%` }"
+                :class="[item.color, item.percentage <= 0 ? 'w-0' : `w-[${item.percentage}%]`]"
```

Comment:
Is this working in production builds? Might be that windi / tailwind would not be able to determine the necessary classes to bundle beforehand.

Reply:
Where is the difference between production builds/dev compose stack?

Reply:
Ok got it. Yes you are right this cant work.

Reply:
Reverted. While we could use workarounds like 

```
<script lang="ts" setup>
const getStatWidthClass = computed(() => {
  return (percentage: number) => {
    if (percentage <= 0) return 'w-0'
    if (percentage <= 5) return 'w-[5%]'
    if (percentage <= 10) return 'w-[10%]'
    if (percentage <= 15) return 'w-[15%]'
    // ... continue for each 5% increment
    if (percentage <= 95) return 'w-[95%]'
    return 'w-full'
  }
})
</script>
```

this seems to be way more complicated and adds a lot of maintenance overhead instead of using inline style for this functionality.